sale_automatic_workflow_job/tests/test_auto_workflow_job.py

```diff
@@ -150,6 +151,7 @@ def test_deleted_sale_order(self):
                 args=(
                     self.sale,
                     safe_eval(workflow.order_filter_id.domain) + workflow_domain,
+                    False,
```

Comment:
```suggestion
                    send_order_confirmation=False,
```

Reply:
I didn't ask for this as I feared you would point at me as "nitpicking" :rofl: